docs/03-04-function-configuration-file.md

```diff
@@ -0,0 +1,193 @@
+| **source.url**             | No | Function | | Provides the address to the Git repository with the Function's code and dependencies. Depending on whether the repository is public or private and what authentication method is used to access it, the URL must start with the `http(s)`, `git`, or `ssh` prefix, and end with the `.git` suffix.  |
+| **source.repository**             | No | Function | Function name | Specifies the name of the Git repository.  |
+| **source.reference**             | No | Function | | Specifies either the branch name or the commit revision from which the Function Controller automatically fetches the changes in Function's code and dependencies.  |
+| **source.baseDir**             | No | Function | | Specifies the location of your code dependencies in the repository. It is recommended to keep the source files at the root of your repository (`/`).  |
+| **source.credentialsSecretName**             | No | Function | | Specifies the name of the Secret with credentials to the Git repository. It is used by the Function Controller to authenticate to the Git repository to fetch the Function's source code and dependencies. This Secret must be stored in the same Namespace as the [GitRepository CR](https://kyma-project.io/docs/main/components/serverless/#custom-resource-git-repository). |
+| **resources**             | No | Function | | Defines CPU and memory available for the Function's Pod to use.  |
+| **resources.limits**             | No | Function | | Defines the maximum available CPU and memory values for the Function.  |
+| **resources.limits.cpu**             | No | Function | `100m` | Defines the maximum available CPU value for the Function.  |
+| **resources.limits.memory**             | No | Function | `128Mi` | Defines the maximum available memory value for the Function.  |
+| **resources.requests**              | No | Function | | Defines the minimum requested CPU and memory values for a Function.  |
+| **resources.requests.cpu**              | No | Function | `50m` | Defines the minimum requested CPU value for the Function.  |
+| **resources.requests.memory**              | No | Function | `64Mi` | Defines the minimum requested memory value for the Function.  |
+| **subscriptions**   | No | Subscription | | Defines a Subscriptions by which the Function gets triggered to perform a business logic defined in the Function's source code.  |
+| **subscriptions.name**           |  Yes | Subscription | Function name | Specifies the name of the Subscription custom resource. It takes the name from the Function unless you specify otherwise.    |
+| **subscriptions.protocol**           | Yes  | Subscription | | Defines the rules and formats applied for exchanging messages between the components of a given messaging system. Subscriptions in Kyma CLI use the [NATS](https://docs.nats.io/) messaging protocol by default. Must be set to `""`.         |
+| **subscriptions.filter** | No | Subscription | | xxx |
```

Comment:
```suggestion
| **subscriptions.filter** | No | Subscription | | Defines the filter for the subscriptions. |
```

I'm not sure what the description should be, perhaps the `xxx` means it will be added later? Ignore this comment if so.

Reply:
Sorry..I missed that...thx a lot for spotting it!